events.js

```diff
@@ -265,15 +265,20 @@ EventEmitter.prototype.listeners = function(type) {
   return ret;
 };
 
+EventEmitter.prototype.listenerCount = function(type) {
+  if(this._events) {
+    var evlistener = this._events[type];
```

Comment:
it could be, i took the variable naming and code structure directly from
node-events. i figured it would be better to mirror node itself as much as
possible.

On Tue, Sep 29, 2015 at 9:30 AM, Daniel Cousens <notifications@github.com>
wrote:

> In events.js
> <https://github.com/Gozala/events/pull/21#discussion_r40670621>:
>
> > @@ -265,15 +265,20 @@ EventEmitter.prototype.listeners = function(type) {
> >    return ret;
> >  };
> >
> > +EventEmitter.prototype.listenerCount = function(type) {
> > +  if(this._events) {
> > +    var evlistener = this._events[type];
>
> evlistener could just be listener?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Gozala/events/pull/21/files#r40670621>.
>